lib/cn/kvset_split.c

```diff
@@ -550,6 +561,18 @@ vblocks_split(
             if (err)
                 goto errout;
 
+            perfc_inc(pc, PERFC_RA_CNCOMP_RREQS);
+            perfc_inc(pc, PERFC_RA_CNCOMP_WREQS);
+            if (perfc_ison(pc, PERFC_RA_CNCOMP_RBYTES) || perfc_ison(pc, PERFC_RA_CNCOMP_WBYTES)) {
+                struct mblock_props props = { 0 };
+
+                err = mpool_mblock_props_get(ks->ks_mp, src_mbid, &props);
+                ev(err);
+
+                perfc_add(pc, PERFC_RA_CNCOMP_RBYTES, props.mpr_write_len);
+                perfc_add(pc, PERFC_RA_CNCOMP_WBYTES, props.mpr_write_len);
```

Comment:
Would make more sense to not bother initializing `props` to 0 and then don't increment counters if there was an error getting props.

Reply:
Good point. Will fix.